internal/util/k8s/parameters.go

```diff
@@ -34,6 +35,12 @@ const (
 	volSnapNameKey        = csiParameterPrefix + "volumesnapshot/name"
 	volSnapNamespaceKey   = csiParameterPrefix + "volumesnapshot/namespace"
 	volSnapContentNameKey = csiParameterPrefix + "volumesnapshotcontent/name"
+
+	// CO cluster Key, set on RBD image.
+	coKeyName = csiParameterPrefix + "co/name"
```

Comment:
have you considered `csi.storage.k8s.io/cluster/name`? none of the other objects have their abbreviated forms